src/cli/commands/uploadFile.ts

```diff
@@ -17,7 +17,11 @@ import { createReadStream, statSync } from 'fs';
 
 import { turboCliTags } from '../constants.js';
 import { UploadFileOptions } from '../types.js';
-import { paidByFromOptions, turboFromOptions } from '../utils.js';
+import {
+  getTagsFromOptions,
+  paidByFromOptions,
+  turboFromOptions,
+} from '../utils.js';
 
 export async function uploadFile(options: UploadFileOptions): Promise<void> {
```

Comment:
Should be able to also add the getTagsFromOptions helper onto uploadFolder now, which will apply the tags on every data item in the folder upload